dashboard.md

```diff
@@ -0,0 +1,57 @@
+
+- [wpt-metadata](https://github.com/web-platform-tests/wpt-metadata) labels specific tests with labels like `interop-2022-subgrid`. These labels are added manually after review of the test lists. The easiest way to enumerate all labeled tests is through the [wpt.fyi API](https://wpt.fyi/api/metadata?includeTestLevel=true&product=chrome).
+- [`category-data.json`](https://github.com/web-platform-tests/results-analysis/blob/main/interop-scoring/category-data.json) defines the focus areas for each year as an identifier like `interop-2024-layout` and a set of labels like `interop-2021-flexbox`, `interop-2022-subgrid`, etc.
+- [`interop-data.json`](https://github.com/web-platform-tests/wpt.fyi/blob/main/webapp/static/interop-data.json) provides human readable names for the focus area identifiers, such as "Layout" for `interop-2024-layout`. This file also includes investigation efforts. The same data is available in [`interop-data.js`](https://wpt.fyi/components/interop-data.js).
+
+## Focus Area Scoring
+
+Scores for each focus area are computed in [results-analysis](https://github.com/web-platform-tests/results-analysis) and published as CSV ([example](https://raw.githubusercontent.com/web-platform-tests/results-analysis/gh-pages/data/interop-2024/interop-2024-experimental-v2.csv)) to GitHub pages. The CSV data and `interop-data.js` is fetched by the wpt.fyi frontend to populate the scores. 
+
+## End-to-End Pipeline
+
+The pipeline for updating scores is as follows:
+
+- All of WPT is run on Chrome, Edge, Firefox, and Safari across two different CI systems: Taskcluster and Azure Pipelines. The cadence varies:
+
+  - Chrome Canary: [every commit](https://github.com/web-platform-tests/wpt/blob/6f491c45bc5182275419be89a0820c2deaddc6b0/tools/ci/tc/tasks/test.yml#L188-L193)
+  - Chrome Stable: [daily](https://github.com/web-platform-tests/wpt/blob/6f491c45bc5182275419be89a0820c2deaddc6b0/tools/ci/tc/tasks/test.yml#L206-L211)
+  - Edge Dev: [every 3 hours](https://github.com/web-platform-tests/wpt/blob/6f491c45bc5182275419be89a0820c2deaddc6b0/.azure-pipelines.yml#L380)
+  - Edge Stable: [daily](https://github.com/web-platform-tests/wpt/blob/6f491c45bc5182275419be89a0820c2deaddc6b0/.azure-pipelines.yml#L344)
+  - Firefox Nightly: [every commit](https://github.com/web-platform-tests/wpt/blob/6f491c45bc5182275419be89a0820c2deaddc6b0/tools/ci/tc/tasks/test.yml#L163-L168)
+  - Firefox Stable: [daily](https://github.com/web-platform-tests/wpt/blob/6f491c45bc5182275419be89a0820c2deaddc6b0/tools/ci/tc/tasks/test.yml#L175-L180)
+  - Safari Technology Preview: [every 3 hours](https://github.com/web-platform-tests/wpt/blob/6f491c45bc5182275419be89a0820c2deaddc6b0/.azure-pipelines.yml#L491)
+  - Safari Stable: [daily](https://github.com/web-platform-tests/wpt/blob/6f491c45bc5182275419be89a0820c2deaddc6b0/.azure-pipelines.yml#L451)
+
+  Runs take 1-3 hours to complete. Results are uploaded to wpt.fyi and are made available within minutes of completion.
```

Comment:
This makes it sound like results are pushed to wpt.fyi by the CI, but I think it's actually something pull-based on the wpt.fyi side?

Reply:
There's some variation between CI systems here.

- There is an API to upload results directly with a username and password, which is used by Deno, Chromium's CI, and probably others.
- For Azure Pipelines there's a [webhook invoked in the CI job](https://github.com/web-platform-tests/wpt/blob/master/tools/ci/azure/fyi_hook.yml) which causes wpt.fyi to download the results from an artifacts API.
- For Taskcluster https://wpt.fyi/api/webhook/taskcluster is invoked as a GitHub webhook, and wpt.fyi has some logic to figure out if all the tasks have finished or not.

The webhook model means that wpt.fyi has knowledge of the CI systems, but has the benefit that no secret is needed, since wpt.fyi doesn't need to trust anything in the webhook, it's just a ping to check for results.